src/common/Color.h

```diff
@@ -256,6 +276,20 @@ class BasicColor
 		data_[ 3 ] = v;
 	}
 
+	CONSTEXPR_FUNCTION_RELAXED component_type ConvertFromSRGB( component_type v ) NOEXCEPT
+	{
+		float f = float( v ) / 255.0f;
+		f = convertFromSRGB( f );
+		return component_type( f * 255 );
+	}
+
+	CONSTEXPR_FUNCTION_RELAXED void ConvertFromSRGB() NOEXCEPT
```

Comment:
It would better if this returns a value instead of mutating. Then it can be used in more places without causing awkwardness.

Reply:
Done.

I did it the mutating way because for example the `Clamp()` function was mutating, but I don't mind.